WeakAuras/GenericTrigger.lua

```diff
@@ -729,6 +729,9 @@ function WeakAuras.ScanEventsInternal(event_list, event, arg1, arg2, ... )
     local updateTriggerState = false;
     for triggernum, data in pairs(triggers) do
       local allStates = WeakAuras.GetTriggerStateForTrigger(id, triggernum);
+      if event == "CURRENT_SPELL_CAST_CHANGED" then -- hack for Cast trigger latency
```

Comment:
The way this is usually done is:
1) Make a fake event WA_EVENT
2) Create a frame watching for CURRENT_SPELL_CAST_CHANGED
3) Which calls ScanEvents with WA_EVENT, "player"

See for example the UNIT_PET -> PET_UPDATE code

Reply:
pushed an update fixing that thank you